docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -28,7 +28,39 @@ A data node also holds various properties and attributes accessible through the
 - _**edit_in_progress**_: The boolean flag signals if the data node is locked for modification.
 - _**properties**_: The dictionary of additional arguments.
 
-# Get data node
+# Create a data node
+
+There are 2 methods to create data nodes in Taipy.
+
+The first method is by creating a scenario via the `taipy.create_scenario()^` method. This method takes a scenario
+configuration as a parameter and returns the created scenario, and all data nodes that are related to the scenario will
+be created alongside the scenario.
+
+The second method is creating a global data node directly using the `taipy.create_global_data_node()^` method.
+This method takes a data node configuration as a parameter and returns the created data node.
+
+This method is useful when you want to create a data node that is independent of any cycle or scenario. For example,
+you can create a data node that stores the dataset for training a model, modify the data node independently, and then
+use this data node in multiple scenarios.
+
+!!! Example
+
+    ```python linenums="1"
+    import taipy as tp
+    from my_config import sales_history_cfg
+
+    sale_history_datanode = tp.create_global_data_node(sales_history_cfg)
+    ```
+
+!!! Warning
+
+    The `taipy.create_global_data_node()^` method only accepts data node configuration with `GLOBAL` scope. If
+    a data node configuration with a different scope is provided, the method will raise the
+    `DataNodeConfigIsNotGlobal^` exception.
```

Comment:
```suggestion
    The `taipy.create_global_data_node()^` method only accepts data node configuration with `GLOBAL` scope. If
    a data node configuration with a different size is provided, the process will raise the
    `DataNodeConfigIsNotGlobal^` exception.
```

Reply:
`Scope` is actually a term in Taipy, so I would not change it.

Reply:
> `Scope` is actually a term in Taipy, so I would not change it.

OK, then we should keep Scope :)